BDEPEND=python_targets_python3_9? ( >=dev-lang/python-3.9.16-r1:3.9 ) python_targets_python3_10? ( >=dev-lang/python-3.10.9-r1:3.10 ) python_targets_python3_11? ( >=dev-lang/python-3.11.1-r1:3.11 ) >=dev-python/gpep517-13[python_targets_python3_9(-)?,python_targets_python3_10(-)?,python_targets_python3_11(-)?]
DEFINED_PHASES=compile configure install prepare test unpack
DESCRIPTION=As the python exception API is quite a bit obscure this python module wraps around python exceptions to provide a clean interface for analysis and logging purposes.
EAPI=8
INHERIT=python-r1 gs-pypi
IUSE=python_targets_python3_9 python_targets_python3_10 python_targets_python3_11
KEYWORDS=~amd64 ~x86
LICENSE=Apache-2.0
RDEPEND=python_targets_python3_9? ( >=dev-lang/python-3.9.16-r1:3.9 ) python_targets_python3_10? ( >=dev-lang/python-3.10.9-r1:3.10 ) python_targets_python3_11? ( >=dev-lang/python-3.11.1-r1:3.11 )
REQUIRED_USE=|| ( python_targets_python3_9 python_targets_python3_10 python_targets_python3_11 )
RESTRICT=test
SLOT=0
SRC_URI=https://files.pythonhosted.org/packages/27/f0/1076576e5c4ed05bac810484c0f50cbf682a804d1d89db50784f0281a7e5/jk_exceptionhelper-0.2021.12.14.1.tar.gz
_eclasses_=out-of-source-utils	1a9007554652a6e627edbccb3c25a439	multibuild	bddcb51b74f4a76724ff7cf8e7388869	multiprocessing	b4e253ab22cef7b1085e9b67c7a3b730	toolchain-funcs	14a8ae365191b518fad51caad7a08f3e	multilib	d1408425c7c4a7669b9b17735404b693	python-utils-r1	429bfd4e8d8e39db5385ba5744f30788	python-r1	3c6cd0f418ba702c186a9865b85e704d	ninja-utils	76050953ad5b70d7e09a6ca55558db92	gs-distutils-r1	d318092e5a4fd070d9e4ffcefb3d054f	g-sorcery	692c9ad583c66d03e7c0e8b3b106a435	gs-pypi	5d35493d186e30ccb1242b91abc6a312
_md5_=87ab92e12984a9a937688f4327dfc73f
